lib/resources/r4/medication_request.yaml

```diff
@@ -131,6 +137,9 @@ fields:
     - display: RxNorm
       system: https://www.nlm.nih.gov/research/umls/rxnorm
       info_link: https://www.nlm.nih.gov/research/umls/rxnorm/index.html
+    - display: Cerner Medication Request Synonym's for orders
+      system: https://fhir.cerner.com/synonym
```

Comment:
@kathypickering  
Display: Cerner Medication Request Synonym's for orders
Fine for the synonym's description

<img width="642" alt="Screenshot 2020-08-20 at 10 39 55 PM" src="https://user-images.githubusercontent.com/3060097/90803348-3c710380-e336-11ea-8442-a6a529b2a365.png">